kolibri/plugins/epub_viewer/assets/src/views/EpubRendererIndex.vue

```diff
@@ -753,17 +753,13 @@
   .epub-renderer {
     position: relative;
     max-height: 100%;
-    padding-top: calc(100% * 8.5 / 11);
+    padding-top: calc(100vh - #{$bottom-bar-height});
```

Comment:
caniuse says this is OK, so I think this is OK: https://caniuse.com/viewport-units

Reply:
On mobile devices, you probably won't see the bottom bar until scrolling up, because it takes into account the address bar. The `100vh` is usually computed with the address bar hidden. This isn't really testable except on an actual mobile device.

Reply:
Thanks @bjester! That's a good catch. @radinamatic I will keep an eye out for any issues you or Peter might run into with QA, and can make adjustments as needed.